forester/src/nullifier/address/processor.rs

```diff
@@ -280,9 +282,14 @@ pub async fn update_merkle_tree<R: RpcConnection>(
     changelog_index: u16,
     indexed_changelog_index: u16,
 ) -> Result<bool, ForesterError> {
+    let start = Instant::now();
     debug!("changelog_index: {:?}", changelog_index);
     debug!("indexed_changelog_index: {:?}", indexed_changelog_index);
 
+    // let (onchain_changelog_index, onchain_indexed_changelog_index) = get_address_account_changelog_indices(&address_merkle_tree_pubkey, &mut *rpc.lock().await).await?;
+    // info!("onchain changelog_index: {:?}", onchain_changelog_index);
+    // info!("onchain indexed_changelog_index: {:?}", onchain_indexed_changelog_index);
```

Comment:
leftovers?

Reply:
Not really, it shouldn't be run in production, but I use these lines quite often for debugging, so it's for convenience.